erizo/src/erizo/rtp/QualityManager.cpp

```diff
@@ -159,31 +161,36 @@ void QualityManager::selectLayer(bool try_higher_layers) {
         aux_temporal_layer++;
       }
     } else {
-      ELOG_DEBUG("message: Skipping below min spatial layer, aux_layer: %d, min_valid_spatial_layer",
+      ELOG_DEBUG("message: Skipping below min spatial layer, aux_layer: %d, min_valid_spatial_layer: %d",
           aux_spatial_layer, min_valid_spatial_layer);
     }
     aux_temporal_layer = 0;
     aux_spatial_layer++;
   }
 
-  if (below_min_layer != slideshow_fallback_active_) {
+  ELOG_DEBUG("below_min_layer %u, freeze_fallback_active_: %u", below_min_layer, freeze_fallback_active_);
```

Comment:
nick: I know we don't follow a strict patter for DEBUG messages, but all the other messages in this class start with `message:`